yarn.lock

```diff
@@ -2924,6 +2924,20 @@
   resolved "https://registry.yarnpkg.com/@hookform/error-message/-/error-message-0.0.5.tgz#606912b506513669980c440f383ff6c78a7aca3a"
   integrity sha512-es7eLLFA3SXNYAT8aUjvf7Gok1eMHK+9DMILtJA7ZEwYZlCCCPifhpoZmY+5SOopEtF7e+qxFOjX+MJnI3jOcg==
 
+"@hugsmidjan/regulations-editor@0.2.0":
+  version "0.2.0"
+  resolved "https://registry.yarnpkg.com/@hugsmidjan/regulations-editor/-/regulations-editor-0.2.0.tgz#92e4ad834bf047206e9d5a8b4a6975e572502262"
+  integrity sha512-LhUnJH8+g30YE8lVCsXdI20opol9rPQfl7bT//PtLpP8ycWzwam+F0Gna0Dr76+khElAx9WMLkUfchPh9HXNzw==
+  dependencies:
+    "@tinymce/tinymce-react" "^3.8.4"
+    cleanup-prettier "npm:prettier@2.2.0"
+    htmldiff-js "^1.0.5"
+    jsdom "^16.4.0"
+    lodash "^4.17.19"
+    qj "npm:@hugsmidjan/qj@^4.6.1"
```

Comment:
Why is `@hugsmidjan/qj` aliased as `qj`?

Reply:
That's a mistake. Should have removed the alias before refactoring into a published lib.

Reply:
Fixed, alias removed